TAF/TestAutomationFramework/Browser/Browser.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System.Collections.ObjectModel;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using TAF.Helper;
+using TAF.Utilities.Logger;
+
+namespace TAF.Core.Browser
+{
+    public class Browser
+    {
```

Comment:
Add ImplicitWait implementation and use it.

Reply:
Resolved